data/database/src/main/java/io/filmtime/data/database/StringListConverter.kt

```diff
@@ -0,0 +1,16 @@
+package io.filmtime.data.database
+
+import androidx.room.TypeConverter
+
+class StringListConverter {
+
+  @TypeConverter
+  fun stringToList(stringList: String): List<String> {
+    return stringList.split(",")
+  }
+
+  @TypeConverter
+  fun listToString(list: List<String>): String {
+    return list.joinToString(",")
+  }
+}
```

Comment:
It may seem like I'm splitting hairs, but this approach has some corner cases. e.g: You try to store 
```kotlin
listOf("A,B", "C", "D")
```

and you restore as:
```kotlin
listOf("A", "B", "C", "D")
```

**No need to change it right now.**